image_client/client_tools.py

```diff
@@ -94,6 +95,10 @@ def main(args):
         if args.collection == "Botany":
             purger = BotanyPurger()
             purger.purge()
+        if args.collection == "Botany_PIC":
+            md5_insert = args.md5
+            PicturaeUndoBatch(MD5=md5_insert)
```

Comment:
able to execute PictuareUndoBatch from command line with new optional argument inputing MD5